trade-finance-manager-ui/templates/case/cancellation/check-details.njk

```diff
@@ -14,51 +14,51 @@
 {% set rowLinkClasses = "govuk-!-padding-top-6 govuk-!-padding-bottom-6 govuk-!-text-align-right" %}
 
 {% set reasonForCancellingUrl %}
-  <a
+<a
     class="govuk-link"
     href="/case/{{ dealId }}/cancellation/reason?status=change"
     data-cy="reason-link">
     Change <span class="govuk-visually-hidden">reason</span>
-  </a>
+</a>
 {% endset %}
 {% set reasonForCancelling =
   [
     { text: 'Reason for cancelling deal', attributes: { "data-cy": "reason-heading" }, classes: rowTextClasses },
-    { text: reason | dashIfEmpty, attributes: { "data-cy": "reason-response" }, classes: rowTextClasses },
+    { text: cancellation.reason | dashIfEmpty, attributes: { "data-cy": "reason-response" }, classes: rowTextClasses },
     { html: reasonForCancellingUrl, classes: rowLinkClasses }
   ]
 %}
 {% set optionsWithHTML = (optionsWithHTML.push(reasonForCancelling), optionsWithHTML) %}
 
 {% set bankRequestDateUrl %}
-  <a
+<a
     class="govuk-link"
     href="/case/{{ dealId }}/cancellation/bank-request-date?status=change"
     data-cy="bank-request-date-link">
     Change <span class="govuk-visually-hidden">bank request date</span>
-  </a>
+</a>
 {% endset %}
 {% set bankRequestDate =
   [
     { text: 'Bank request date', attributes: { "data-cy": "bank-request-date-heading" }, classes: rowTextClasses},
-    { text: bankRequestDate | dashIfEmpty, attributes: { "data-cy": "bank-request-date-response" }, classes: rowTextClasses },
+    { text: cancellation.bankRequestDate | formatUnixTimestamp('d MMMM yyyy') | dashIfEmpty, attributes: { "data-cy": "bank-request-date-response" }, classes: rowTextClasses },
     { html: bankRequestDateUrl, classes: rowLinkClasses }
   ]
 %}
 {% set optionsWithHTML = (optionsWithHTML.push(bankRequestDate), optionsWithHTML) %}
 
 {% set effectiveFromDateUrl %}
-  <a
+<a
     class="govuk-link"
     href="/case/{{ dealId }}/cancellation/effective-from-date?status=change"
     data-cy="effective-from-link">
     Change <span class="govuk-visually-hidden">date effective from</span>
-  </a>
+</a>
 {% endset %}
-  {% set effectiveFromDate =
+{% set effectiveFromDate =
     [
       { text: 'Date effective from', attributes: { "data-cy": "effective-from-heading" }, classes: rowTextClasses },
-      { text: effectiveFromDate | dashIfEmpty, attributes: { "data-cy": "effective-from-response" }, classes: rowTextClasses },
+      { text: cancellation.effectiveFrom | formatUnixTimestamp('d MMMM yyyy') | dashIfEmpty, attributes: { "data-cy": "effective-from-response" }, classes: rowTextClasses },
```

Comment:
Good change, I prefer the formatting handled here - thanks!

Reply:
Thought it made sense when I need the hidden cancellation input to only pass this data in once :)